Doc/library/os.rst

```diff
@@ -1195,6 +1195,32 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    .. versionadded:: 3.3
 
 
+.. function:: preadv(fd, buffers, offset, flags = None)
```

Comment:
I don't know the answer to this, but should it be `flags=None` with no spaces?

Reply:
You are right. The `None` was because a previous declaration before switching to the default suggested by @njsmith. It should be `flags=0`. Corrected in 1988d1a.